python/hopsworks_common/core/services_api.py

```diff
@@ -18,10 +18,11 @@
 
 from typing import Any, Dict
 
-from hopsworks_common import client
+from hopsworks_common import client, usage
 
 
 class ServicesApi:
+    @usage.method_logger
     def get_service(self, service: str) -> Dict[str, Any]:
```

Comment:
Same here. I think we can remove the usage logger, and also I can't find where this method is used or how users can access it without instantiating the ServiceApi class by themselves.